resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -28,7 +28,6 @@ ARG spark_uid=185
 # docker build -t spark:latest -f kubernetes/dockerfiles/spark/Dockerfile .
 
 RUN set -ex && \
-    sed -i 's/http:\/\/deb.\(.*\)/https:\/\/deb.\1/g' /etc/apt/sources.list && \
```

Comment:
Note:
- OpenJDK is built on top of Debian, Dockerfile was modify http to https of its [sources list file](https://wiki.debian.org/SourcesList#Example_sources.list), where Eclipse temurin is built on top of [Ubuntu](https://manpages.ubuntu.com/manpages/xenial/man5/sources.list.5.html)